Plausible/Testable.lean

```diff
@@ -5,6 +5,7 @@ Authors: Henrik Böving, Simon Hudon
 -/
 import Lean.Elab.Tactic.Config
 import Plausible.Sampleable
+import Batteries
```

Comment:
What particular functionality are you pulling in batteries here for?

Also note the remark about not importing an entire library at once from above.